mobx/lib/src/api/annotations.dart

```diff
@@ -1,12 +1,19 @@
 /// Internal class only used for code-generation with `mobx_codegen`.
 ///
-/// During code-generation, this type is detected to identify a MobX Store class
-class MakeStore {
-  const MakeStore._();
+/// During code-generation, this type is detected to identify Store class with generated toString() method
+class MakeStoreConfig {
+  const MakeStoreConfig({this.hasToString = true});
+  const MakeStoreConfig._({this.hasToString = true});
+  final bool hasToString;
 }
 
-/// Declares the annotation identifier for a Store class.
-const MakeStore store = MakeStore._();
+class StoreConfig extends MakeStoreConfig {
```

Comment:
Why do we need a subclass? We can just rename `MakeStoreConfig` -> `StoreConfig`?
```suggestion
class StoreConfig {
```